### ISSUE TITLE
Should create corpus directory with the creation of fuzz directory

### ISSUE THREAD

**Issue by nagisa** [rust-fuzz/cargo-fuzz#30]

**Comment by Manishearth** [comment]
Not exactly. `cargo fuzz --init` is something you run once, check in, and no other contributor runs again. Since the directory is gitignored, it can't be checked in. This is why we do this at fuzz-time.

We could still create it at init time, but we should also create it at fuzz time.

**Comment by Manishearth** [comment]
Yeah. So we should create it in both fuzz and init.
